lib/ops/opsservice/versions.go

```diff
@@ -255,3 +255,12 @@ const (
 This cluster image does not contain required intermediate runtimes.
 Please rebuild it as described in https://gravitational.com/gravity/docs/cluster/#direct-upgrades-from-older-lts-versions.`
 )
+
+var (
+	// TeleportBrokenJoinTokenVersion is version of the release affected by
+	// the issue with Teleport using incorrect auth token on joined nodes.
+	//
+	// Github issue: https://github.com/gravitational/gravity/issues/1445.
+	// KB: https://community.gravitational.com/t/recover-teleport-nodes-failing-to-join-due-to-bad-token/649.
+	TeleportBrokenJoinTokenVersion = semver.New("5.5.40")
```

Comment:
This won't matter for master as long as we have not ported intermediate upgrades?

Reply:
Yeah, probably doesn't make sense to have this specific check for anything beyond 7.0. I'll remove it from master's port.